gwt-event-compat-gwt2-tests/pom.xml

```diff
@@ -89,6 +56,23 @@
       <plugin>
         <groupId>com.mycila</groupId>
         <artifactId>license-maven-plugin</artifactId>
+        <version>${maven.license.plugin}</version>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-install-plugin</artifactId>
+        <version>${maven.install.plugin}</version>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
+      </plugin>
+      <plugin>
```

Comment:
isnt this inherited from the parent pom?

Reply:
Yes, it is. I also added the install plugin inside the parent. This will avoid the deploy of the parent pom as well.